.github/workflows/ci.yml

```diff
@@ -90,20 +90,40 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Determine tag
+      - name: Determine Docker Tag
         uses: haya14busa/action-cond@v1
         id: imagetag
         with:
           cond: ${{ github.event_name == 'pull_request' }}
           if_true: ${{ github.sha }}
-          if_false: "master"
+          if_false: 'master'
 
-      - name: Build
-        uses: docker/build-push-action@v1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: dexidp/dex
-          tags: ${{ steps.imagetag.outputs.value }}
-          add_git_labels: true
-          push: ${{ github.event_name == 'push' }}
+          platforms: all
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          install: true
+          version: latest
+          driver-opts: image=moby/buildkit:master
```

Comment:
Why do you need the `master` image here specifically?

Reply:
There is a bug that is only fixed on master right now in the github action and this is their temporary work around.

Reply:
Issue ref: https://github.com/docker/buildx/issues/386

Reply:
Got it, thanks!

Reply:
Should we add a TODO here so that we pin it back to a version once a new tag is cut?